### DISCUSSION TITLE
Planned release date v0.26

### DISCUSSION THREAD

**Discussion by gixslayer** [hneemann/Digital#610] [Q&A] [answered]
First of all, in case I haven't said this before, thank you for this amazing program! We use it for a university course since last year, and our experience so far has been very positive. I'm currently involved with doing some prep work before the next iteration of the course starts January 25th, and was wondering if there is already a planned release date for the next version.

There are some features introduced post v0.25 that are very interesting to us, mainly revolving around the improved test cases in the context of processors, which I'd like to use. We have to make a decision soon however on the version we will be targeting this year (either v0.25 or v0.26), as we have to update/adjust some internal tooling and content based on this. It is also relevant because of changes to the K-map and FSM features, both of which will potentially be used as instructional aids (as we too are forced to provide online education/distance learning, and I think something visual and interactive like these Digital features are amazing for this).

I'm hoping it would be possible to have a v0.26 release soon, but at the same time I don't want to tread on any plans you might have regarding this.

**Comment by hneemann** [comment, marked as answer]
There are two differences between releases and pre-releases: One is the running of the GUI tests, which take a long time and therefore are not run on every pre-release. I can run these at any time, of course. The other is the translations. In releases all translations are up to date, in pre-releases it is only German and English.
I will ask the volunteers to update the translations. As soon as they are up to date, I will create a new release.
But if you only need english or german, you can just as well use the pre-release v0.25-253 (see [here](https://infdigital.dhbw-mosbach.de/)). I just successfully ran the GUI tests for v0.25-253.
